pkg/kubectl/resource_printer.go

```diff
@@ -786,8 +788,17 @@ func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 			return err
 		}
 	}
+
+	podColor := "<fg 2>" // green
+	if readyContainers != totalContainers {
+		podColor = "<fg 11>" // yellow
+	}
+	if readyContainers == 0 {
+		podColor = "<fg 1>" // red
```

Comment:
Yeah, that works for me. I'd love to have yellow for "warning" (pending, some restarts recently, etc) and "red" for "not working"

I'll update the pr.